sensu/config.go

```diff
@@ -131,3 +141,17 @@ func (c *Config) Checks() []*check.Check {
 
 	return []*check.Check{}
 }
+
+// Removes duplicates from string slices
+func RemoveDuplicates(xs *[]string) {
```

Comment:
Could you rewrite this function and make it something immutable. `removeDuplicates([]string) []string`

And it does not need to be public, please use a lowercase for the first letter of the function